src/frontend/src/pages/ChangeRequestDetailPage/ReviewChangeRequestView.tsx

```diff
@@ -205,26 +207,24 @@ const ReviewChangeRequestsView: React.FC<ReviewChangeRequestViewProps> = ({
           </form>
         </DialogContent>
         <DialogActions>
-          <Button
-            variant="contained"
-            color="success"
-            type="submit"
-            form="review-notes-form"
-            sx={{ textTransform: 'none', fontSize: 16 }}
-            onClick={() => handleAcceptDeny(true)}
-          >
-            Accept
-          </Button>
-          <Button
+          <NERFailButton
             type="submit"
             form="review-notes-form"
             variant="contained"
-            color="error"
-            sx={{ textTransform: 'none', fontSize: 16 }}
+            sx={{ mx: 1, textTransform: 'none', fontSize: 16 }}
             onClick={() => handleAcceptDeny(false)}
           >
             Deny
-          </Button>
+          </NERFailButton>
+          <NERSuccessButton
+            variant="contained"
+            type="submit"
+            form="review-notes-form"
+            sx={{ mx: 1, textTransform: 'none', fontSize: 16 }}
```

Comment:
I don't think you need textTransform or fontsize, none of the other fail/success buttons are using these. See if it still looks good/standard with the buttons on the new change request form on develop if you remove those two on both buttons.

Reply:
Ok will do, do I edit this on the same branch or make a new branch for it?

Reply:
edit it on the same branch, add and commit the changes, then if you git push again, it'll automatically push the changes to this pull request